start-local.sh

```diff
@@ -93,6 +106,8 @@ startup() {
   kibana_container_name="kibana-local-dev${ES_LOCAL_DIR:+-${ES_LOCAL_DIR}}"
   # Kibana settings container name
   kibana_settings_container_name="kibana-local-settings${ES_LOCAL_DIR:+-${ES_LOCAL_DIR}}"
+  # OTEL container name
+  otel_container_name="otel-collector${ES_LOCAL_DIR:+-${ES_LOCAL_DIR}}"
```

Comment:
Wdyt of renaming the container name to `elastic-otel-collector` or `edot-collector` to differentiate from the upstream image?

Reply:
Sorry, I should have worded that better. My suggestion would be to include the `edot` keyword on the container name prefix:
```suggestion
  otel_container_name="edot-collector${ES_LOCAL_DIR:+-${ES_LOCAL_DIR}}"
```

(EDOT = [Elastic Distributions of OpenTelemetry](https://www.elastic.co/observability-labs/blog/elastic-distributions-opentelemetry))